Sources/SPTPersistentCacheGarbageCollectorScheduler.h

```diff
@@ -21,8 +21,9 @@
 #import <Foundation/Foundation.h>
 
 @class SPTPersistentCache;
+@class SPTPersistentCacheOptions;
 
-@interface SPTPersistentCacheTimerProxy : NSObject
+@interface SPTPersistentCacheGarbageCollectorScheduler : NSObject
```

Comment:
Been thinking, this name implies that the class has two jobs:
- Garbage collection
- Scheduling

How about we rename it to just `SPTPersistentCacheGarbageCollector`?

Then the `-scheduleGarbageCollection` could be named just `-schedule` and still be clear while also being more succinct. The same for `-unscheduleGarbageCollection` which could be just `-unschedule`.

What do you think @iOSCowboy?